main.py

```diff
@@ -1,27 +1,79 @@
 import requests
 import time
+import json
 
+#API_KEY = "API_KEY"
 API_KEY = "API_KEY"
-API_URL = "https://yp1ypp2boj.execute-api.us-east-2.amazonaws.com/prod"
+API_URL = "https://api.censori.dev.ontelio.ai"
 
-# Send media to begin redaction process
-def send_media():
+def redact_transcript():
+    """
+    Redacts a transcript only.
+
+    Raises:
+        e: json body is not in the format of a proper json.
+        e: error in the response body.
+    """
+    #open json file
+    try:
+        with open(f"./files/preamble.json", 'rb') as file:  # Path to file
+            _transcript = json.load(file)
+    except Exception as e:
+        raise e
+    
+    upload_key_endpoint = f"{API_URL}/redact"
+    # Call to get upload key
+    upload_key_json_body = {
+         #"entities": [] # you can leave the entities blank if you want to redact everything.
+        "entities": ["PERSON", "LOCATION"],  # Types of entities to be redacted
```

Comment:
Can you add an address where they can find the list of entity types? We should probably add that to a README here too.